src/Options/Scales/Ticks.php

```diff
@@ -441,6 +446,26 @@ public function setReverse( $reverse )
 	}
 
 	/**
+	 * @return int
+	 */
+	public function getMax()
+	{
+		return $this->max;
+	}
+
+	/**
+	 * @param int $max
+	 *
+	 * @return $this
+	 */
+	public function setMax($max)
+	{
+		$this->max = $max;
```

Comment:
One last thing. If you change

```php
$this->max = $max;
```
to
```php
$this->max = intval($max);
```
we're done.

Reply:
Sorry for the late reply, I haven't seen the notification. I added intval and also updated the code style to comply with your's.